package.json

```diff
@@ -9,7 +9,8 @@
     "dev-frontend": "parcel watch src/frontend/*.html",
     "dev-backend": "nodemon --watch src/backend src/backend",
     "prebuild": "rm -Rf dist/*.js dist/*.css dist/*.map",
-    "build": "npm test && npm run prebuild && parcel build src/frontend/*.html"
+    "build": "npm test && npm run prebuild && parcel build src/frontend/*.html",
+    "electron": "electron ./src/backend/electron.js"
```

Comment:
How are the parameters (like `--notebook` you mention in the README for electron) passed to the script? Is it transparent for npm ?

Reply:
electron.js immediately calls the original index.js express app which is where the parameters are parsed out, electron isn't touching them.